app/Models/Flights.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Flights extends Model
```

Comment:
```suggestion
class Flight extends Model
```
đổi cả tên file nhé, có một số quy tắc đặt tên trong laravel bạn tham khảo nhé https://webdevetc.com/blog/laravel-naming-conventions/#section_naming-models-in-laravel